plugins/dynamic-home-page/src/components/Markdown.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import { MarkdownContent } from '@backstage/core-components';
+
+import { makeStyles } from '@material-ui/core/styles';
```

Comment:
Use `makeStyles` from `tss-react/mui` instead, like [this](https://github.com/janus-idp/backstage-showcase/blob/main/packages/app/src/components/catalog/Grid/Grid.tsx#L3-L13) for example.

Reply:
I like to keep this aligned with the MUI v4 components that are used in the plugin. For now. I will add a comment to our tech debt issue to update to v5 to check for makeStyles from Mui v4.